commerce-notification-service/src/main/java/com/liferay/commerce/notification/internal/util/CommerceNotificationHelperImpl.java

```diff
@@ -182,99 +158,48 @@ protected void sendNotification(
 				siteDefaultLocale);
 		}
 
-		_commerceNotificationQueueEntryLocalService.
-			addCommerceNotificationQueueEntry(
-				user.getUserId(), groupId,
-				commerceNotificationType.getClassName(object),
-				commerceNotificationType.getClassPK(object),
-				commerceNotificationTemplate.
-					getCommerceNotificationTemplateId(),
-				commerceNotificationTemplate.getFrom(), fromName,
-				user.getEmailAddress(), user.getFullName(),
-				commerceNotificationTemplate.getCc(),
-				commerceNotificationTemplate.getBcc(), subject, body, 0);
-	}
-
-	protected void sendNotificationsToAccountGroupUsers(
-			long[] commerceAccountGroupIds,
-			CommerceNotificationTemplate commerceNotificationTemplate,
-			CommerceNotificationType commerceNotificationType, Object object)
-		throws PortalException {
-
-		List<Long> commerceAccountUserIds =
-			_commerceAccountGroupLocalService.
-				getCommerceAccountUserIdsFromAccountGroupIds(
-					commerceAccountGroupIds, QueryUtil.ALL_POS,
-					QueryUtil.ALL_POS);
-
-		long[] commerceAccountUserIdsArray = ArrayUtil.toLongArray(
-			commerceAccountUserIds);
-
-		for (long userId : commerceAccountUserIdsArray) {
-			sendNotification(
-				commerceNotificationTemplate, commerceNotificationType, object,
-				userId);
-		}
-	}
-
-	protected void sendNotificationsToUserIds(
-			long[] userIds, long[] commerceAccountGroupIds,
-			CommerceNotificationTemplate commerceNotificationTemplate,
-			CommerceNotificationType commerceNotificationType, Object object)
-		throws PortalException {
-
-		for (long userId : userIds) {
-			List<CommerceAccountUserRel> commerceAccountUserRelsByUserId =
-				_commerceAccountUserRelLocalService.
-					getCommerceAccountUserRelsByCommerceAccountUserId(userId);
-
-			List<Long> commerceAccountIds = new ArrayList<>();
-
-			for (CommerceAccountUserRel commerceAccountUserRel :
-					commerceAccountUserRelsByUserId) {
-
-				commerceAccountIds.add(
-					commerceAccountUserRel.getCommerceAccountId());
-			}
-
-			long[] commerceAccountIdsArray = ArrayUtil.toLongArray(
-				commerceAccountIds);
-
-			if (ArrayUtil.containsAll(
-					commerceAccountIdsArray, commerceAccountGroupIds)) {
-
-				sendNotification(
-					commerceNotificationTemplate, commerceNotificationType,
-					object, userId);
-			}
+		String to = formatString(
+			commerceNotificationType, commerceNotificationTemplate.getTo(),
+			object, userLocale);
+
+		String[] toUserIds = StringUtil.split(to, ",");
```

Comment:
No need to specify `","` as the delimiter here -- it's the default

Reply:
Actually, even better, pass `GetterUtil.DEFAULT_LONG` as a third parameter, and it'll give you a `long[]` so you don't need to call `GetterUtil.getLong` on each `toUserId` in the loop.

(But if we leave the comma in, it should really be `StringPool.BLANK` rather than a string literal.)